libpod/options.go

```diff
@@ -548,14 +553,108 @@ func WithRuntimeFlags(runtimeFlags []string) RuntimeOption {
 
 // Container Creation Options
 
+// WithCpuSetCpusCtr computes and sets the Cpus linux resource string which determines the amount of cores, from those available,  we are allowed to execute on
+func WithCPUSetCPUsCtrInfra(res string) CtrCreateOption {
```

Comment:
I would prefer to have this just take an array of ints, and have the string parsing be done in the command line - that's generally been our approach with Libpod.

Reply:
Also - why do we need the per-container options here? The container code should not need changing. The fact that they are in the pod cgroup should be sufficient.

Reply:
@mheon is this true for infra as well? If so I can snip a lot of code I just wasn't sure